### PR TITLE
Add appendix

### LINKED ISSUE #11: Missing Appendix
Currently there is no possibility to add large graphics or code examples at the end of the book.
This would be nice as a code example > 1 page seems odd in the book itself.

The requested is focused on but not limited to [this](https://github.com/dhbw-stginf16a/exploring-cassandra/blob/master/sparse_rows)

### PR DESCRIPTION
**Pull request by guserav**
Closes #11

### CODE REVIEW

**Comment by lschuermann** [comment]
I'd rather have it so that the content is merged with the appendix. However, as you're probably merging the content rather soon, this should be okay.

Can you please use the same file name for appendices as for the content file (indicated in the readme)? This makes sorting the inputs in `main.tex` easier.

**Comment by lschuermann** [comment]
Adding @danielrutz as a reviewer, as this is a fundamental change.

**Review by danielrutz — APPROVED** [review]
I do not see a fundamental problem here. Please wait for @lschuermann to approve before merging.

**Review by lschuermann — APPROVED** [review]

**Pull request merged**
